### ISSUE TITLE
Discrepancies between IDE mode protocol and Emacs idris-mode

### ISSUE THREAD

**Issue by abailly** [idris-lang/Idris2#97]
Official IDE mode protocol documentation can be found at http://docs.idris-lang.org/en/latest/reference/ide-protocol.html
It seems the actual [idris-mode](https://github.com/idris-hackers/idris-mode) implementation is somehow not in sync with this document:

* [ ] it sends `(:interpret ":consolewidth 100")` to set the layout limit
* [ ] it sends `(:get-options)` to retrieve the backend's options

**Comment by ohad** [comment]
OK, the `:consolewidth 100` is an Idris1 REPL command that's not implemented currently in Idris2.
@edwinb: do we want to support `:consolewidth` in the future, or is this a bug with `idris-mode`?

**Comment by edwinb** [comment]
We probably should support `:consolewidth`. It was used in Idris 1 to help with pretty printing and I can imagine wanting to do something similar.

**Comment by jfdm** [comment]
Indeed, there was a recent port of Haskell's pretty-printer to Idris2 (still a PR #393 ) and if Idris2's display internals are prettified then this would be useful for compliance....
